modules/document_repository/jsx/editForm.js

```diff
@@ -116,35 +121,36 @@ class DocEditForm extends React.Component {
           />
           <SelectElement
             name="instrument"
-            label="Instrument"
+            label={t('Instrument', {ns: 'document_repository'})}
```

Comment:
not sure if it currently exists in the loris namespace but "Instrument" is a common loris term that should be in that namespace.